exporter_test.go

```diff
@@ -816,6 +816,51 @@ type = "Apache-2.0"
 				assertLogEntry(t, logHandler, "Adding 1/1 app layer(s)")
 			})
 
+			when("app dir has ending path separator", func() {
```

Comment:
Now that we have coverage in the archive package tests, I think we should remove this test and "app dir has dot element". It seems like a lower level concern and these higher level exporter tests are more trouble to maintain.